article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -245,6 +245,15 @@ object DotcomponentsDataModel {
       }
     }
 
+    def buildFullCommercialUrl(bundlePath: String): String = {
```

Comment:
It feels like this could be used more generally, although maybe it's okay to keep it commercial-bundle-specific until more use cases present themselves. I'm surprised this need has not arisen before though!

Reply:
I toooooootally agree with you :)